lms/djangoapps/instructor/views/api.py

```diff
@@ -987,16 +988,16 @@ def extract_user_info(user):
         }
         if course_cohort is not None and rolename == 'staff':
             assigment_checker = models.Case(
-                models.When(cohort_mangers__contains=request.user.id,then=models.Value(True)),
+                models.When(
+                    cohortassigment__in=CohortAssigment.objects.filter(user=user).all() ,then=models.Value(True)
```

Comment:
minor nit: s/`all() ,then`/`all(), then`/
Why do `.all()` is used after the `.filter()` it will return required queryset, doesn't it?

Reply:
As for me, it is more explicitly. As all variant of the query is identical SQL request I prefer left it as is.